### PR TITLE
Move initializer list to declaration block

### PR DESCRIPTION
**Pull request by tobiolo**

### CODE REVIEW

File: src/myframe.h [hunk 1]
```diff
@@ -21,9 +21,9 @@ struct MyFrame : wxFrame {
                  // device-independent pixels
     std::vector<std::string> scripts_in_menu;
     bool zenmode;
-    ColorDropdown *celldd = nullptr;
```
Comment (aardappel) [inline src/myframe.h:24]: Initialization directly in the declaration is superior to doing it in the initializer list when possible, so if anything, better to move other fields to the declaration where possible.

**Comment by tobiolo** [comment]
Is this style ok for you?

File: src/myframe.h (hunk 1 above)
Comment (tobiolo) [reply src/myframe.h:24]: Thanks for the hint! This saves a lot of lines.

File: src/document.h [hunk 2]
```diff
@@ -6,28 +6,26 @@ struct UndoItem {
     Vector<Selection> path, selpath;
     Selection sel;
     unique_ptr<Cell> clone;
-    size_t estimated_size;
+    size_t estimated_size{ 0 };
```
Comment (aardappel) [inline src/document.h:9]: I would just have done `= 0` like we already have in some places. If there's a reason `{}` is better, I'd put a space ahead of the first `{`.

**Pull request merged**